packages/notifi-react-card/lib/components/subscription/defaults.css

```diff
@@ -635,7 +635,7 @@
 
 .NotifiIntercomHeader__content {
   font-size: 14px;
-  font-weight: 800;
+  font-weight: 650;
```

Comment:
Can you please check on the `font-weight`? 

The font in the designs in the Figma do look even more thin. 

<img width="400" alt="Screen Shot 2022-12-01 at 10 50 49 AM" src="https://user-images.githubusercontent.com/108922885/205135752-e92e9d7f-81af-41e0-8508-66ad8e7ba51d.png">

Reply:
the font-weight in design is 800, which is I used and looks much thick than figma, and with 600, it is thinner than in figma, so I changed to 650, not exactly the same but similar. here is the screen shot for font-weight 600:
![Screenshot 2022-12-01 at 11 08 12 AM](https://user-images.githubusercontent.com/26240001/205142329-a18040fb-6ee7-41b7-83a7-c3e12bfb1c41.png)

Reply:
I think 600 does look better. Maybe send this screenshot to Sam and see what she thinks?

Reply:
updated with 600 and also send a screen shot to Sam, will merge this in for now and if we need to change font weight, I'll create another PR. but I think it does look better too:)